google-mysql.yml

```diff
@@ -141,6 +141,31 @@ provision:
     constraints:
       minimum: 0
       maximum: 7
+  - field_name: highly_available
+    type: boolean
+    details: |
+      Enable regional high availability for the service instance. Requires transaction log backups to be enabled. See
+      https://cloud.google.com/sql/docs/mysql/high-availability for more details.
```

Comment:
question: I just read that to enable HA **automatic backups** and binary logs have to be enabled. We enable the backups by setting the `backups_retain_number` property to 7, but it would be nice to mention it in the documentation. Do you think it would be appropriate to add an explanatory note?